src/components/home/Categories.jsx

```diff
@@ -64,7 +64,7 @@ const CATEGORIES_INFO = [
     img: sterhoscopeImg,
   },
   {
-    name: "Nutrition",
+    name: "NUTRITION",
     img: NutritionImg,
```

Comment:
Instead of changing it here manually, can you please use `text-transform: uppercase` for that className?

Reply:
there is no className for this one?

Reply:
done!